get_ior_results.sh

```diff
@@ -16,57 +16,14 @@ cd $RES_DIR
 
 #Test name
 current=${PWD##*/}
+echo $current
 
-#Output files - raw* for extracting the results from tests and result* for final results in csv format into a file ready for plotting
-raw="./raw_$current"
-result="./result_$current"
-
-process_result() {
-    infile=$1
-    outfile=$2
-    if [ -f ./tmp ] ; then rm ./tmp ; fi
-    echo "Num_Servers,Write(GiB/s),Read(GiB/s),Write(MiB/s),Read(MiB/s)" > $outfile
-    {
-        last=0
-        {
-        read
-        while  read -r line ; do
-            cur=`awk -F, '{print $1}' <<< "$line"`
-            wr=`awk -F, '{print $5}' <<< "$line"`
-            rd=`awk -F, '{print $6}' <<< "$line"`
-            if  [[ $cur -ne $last ]] ; then
-                calc_averages  $cnt  
-                last=$cur
-                cnt=1
-                sum_wr=$wr
-                sum_rd=$rd
-            else
-                let "cnt++"
-                sum_wr=`echo "$sum_wr + $wr" | bc`
-                sum_rd=`echo "$sum_rd + $rd" | bc`
-            fi
-        done
-        }  < "$infile"
-        calc_averages  $cnt 
-        echo $infile $outfile
-    }
-    cat tmp | tee -a  $outfile
-}
-
-calc_averages () {
-    num=$1
-    if [[  $last -ne "0" ]] ; then
-    av_wr=`echo "scale=2;$sum_wr / $num" | bc`
-    av_rd=`echo "scale=2;$sum_rd / $num" | bc`
-    wr_GiB=`echo "scale=2;$av_wr / 1024" | bc`
-    rd_GiB=`echo "scale=2;$av_rd / 1024" | bc`
-    echo "$last","$wr_GiB","$rd_GiB","$av_wr","$av_rd" >> tmp
-    fi
-}
+#Output file
+result="$RES_DIR/result_$current"
```

Comment:
You have cd $RES_DIR on line no.15 so is this going to work?

Reply:
Yes it does work. Essentially using  $RES_DIR/result_$current instead of ./$RES_DIR/result_$current  The intent is to keep the result along in the result directory itself.